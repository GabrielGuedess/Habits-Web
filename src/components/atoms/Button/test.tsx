import { render, screen } from '@testing-library/react';

import { Button } from '.';

describe('<Button />', () => {
  it('should render the Button', () => {
    const { container } = render(<Button />);

    expect(
      screen.getByRole('button', { name: /Novo h√°bito/i }),
    ).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });
});
