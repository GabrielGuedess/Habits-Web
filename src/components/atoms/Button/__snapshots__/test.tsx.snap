// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> should render the Button 1`] = `
<button
  class="group flex items-center gap-3 rounded-lg border border-violet-500 px-6 py-4 font-semibold transition-all hover:border-violet-300"
  type="button"
>
  <svg
    class="text-violet-500 transition-all group-hover:text-violet-300"
    fill="currentColor"
    height="20"
    viewBox="0 0 256 256"
    width="20"
    xmlns="http://www.w3.org/2000/svg"
  >
    <rect
      fill="none"
      height="256"
      width="256"
    />
    <line
      fill="none"
      stroke="currentColor"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="16"
      x1="40"
      x2="216"
      y1="128"
      y2="128"
    />
    <line
      fill="none"
      stroke="currentColor"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="16"
      x1="128"
      x2="128"
      y1="40"
      y2="216"
    />
  </svg>
  Novo h√°bito
</button>
`;
